RadixWallet/Features/InteractionReview/Sections/Accounts/Accounts.swift

```diff
@@ -0,0 +1,127 @@
+import ComposableArchitecture
+import SwiftUI
+
+// MARK: - InteractionReview.Accounts
+extension InteractionReview {
+	@Reducer
+	struct Accounts: Sendable, FeatureReducer {
```

Comment:
I'd prefer if we could avoid calling reducers the same thing as models, makes searches (or possible find replace all) tricker, can it be called `AccountsSection` or similar? I dont feel too strongly about this though, so feel free to ignore

Reply:
I'd prefer sticking as it is. The find & replace is already pretty bad on Xcode, meaning that even if we name it `AccountsSection`, it will still break if we have other `AccountsSections` anywhere else. When I just renamed the `send` case to `submitTransaction`, Xcode wanted to update tests that were doing `viewStore.send()`.

Then, keeping it short-named and enclosed under `InteractionReview` namespace makes it easier to follow IMO.